templates/landscape/install.html

```diff
@@ -182,183 +157,7 @@ <h2>Install Landscape Server to manage machines and containers</h2>
-                  </div>
-                </li>
-                <li class="p-list__item">
-                  <p class="u-no-padding--top">Update HAProxy with your certificate</p>
-                  <div class="p-code-snippet">
-                    <pre class="p-code-snippet__block is-wrapped"><code>juju config haproxy ssl_cert="$FULLCHAIN" ssl_key="$PRIVKEY"</code></pre>
-                  </div>
-                </li>
-              </ol>
-            </div>
-          </div>
-
-          <hr class="u-no-margin--bottom">
-          <div class="col-3 col-medium-1 u-hide--small">
-            <p class="p-step-counter"></p>
-          </div>
-          <div class="col-9 col-medium-5">
-            <div class="p-strip is-shallow u-no-padding--top">
-              <div class="p-strip is-shallow u-no-padding--top">
-                <h3 class="p-heading--2">Configure postfix for email</h3>
-              </div>
-              <p>The SMTP example in Step 1 uses SendGrid as an example, replace it with your own SMTP information</p>
-            </div>
-            <div class="p-strip u-no-padding--top">
-              <ol class="p-list--divided">
-                <li class="p-list__item">
-                  <p class="u-no-padding--top">Set SMTP information in variables</p>
-                  <div class="p-code-snippet">
-                    <pre class="p-code-snippet__block is-wrapped"><code>SMTP_HOST='smtp.sendgrid.net'
-SMTP_PORT='465'
-SMTP_USERNAME='apikey'
-SMTP_PASSWORD='YOUR_API_KEY_GOES_HERE'</code></pre>
-                  </div>
-                </li>
-                <li class="p-list__item">
-                  <p class="u-no-padding--top">Deploy the postfix-relay subordinate charm</p>
-                  <div class="p-code-snippet">
-                    <pre class="p-code-snippet__block is-wrapped"><code>juju deploy postfix-relay --config smtp_auth_password="$SMTP_PASSWORD" --config smtp_auth_username="$SMTP_USERNAME" --config ssl_ca=$(curl -s https://www.thawte.com/roots/thawte_Primary_Root_CA.pem | base64 | tr -d '\n') --config relayhost="[${SMTP_HOST}]:${SMTP_PORT}"</code></pre>
-                  </div>
-                </li>
-                <li class="p-list__item">
-                  <p class="u-no-padding--top">Add charm relations</p>
-                  <div class="p-code-snippet">
-                    <pre class="p-code-snippet__block is-wrapped"><code>juju add-relation postfix-relay landscape-server</code></pre>
-                  </div>
-                </li>
-              </ol>
-            </div>
-          </div>
-        </div>
-      </div>
-
-      <div class="p-tabs__content u-hide" tabindex="0" role="tabpanel" id="quickstart-tab" aria-labelledby="quickstart" style="scroll-margin-top:86px;">
+      <div class="p-tabs__content" tabindex="0" role="tabpanel" id="quickstart-tab" aria-labelledby="quickstart" style="scroll-margin-top:86px;">
```

Comment:
Below we have this line: "sudo apt update && sudo apt install -y wget software-properties-common"
The word "wget" should be removed from there, and it should be replaced with "ca-certificates", it should read:
```
sudo apt update && sudo apt install -y ca-certificates software-properties-common 
```

Reply:
Can you add `ca-certificates` to the copy docs? I believe it's missing too?

Reply:
i checked for it, and do see it in the copydocs:
![image](https://github.com/canonical/ubuntu.com/assets/388881/b8e4bc3d-a6ce-4857-8d17-9ccde22868ea)